sources/Adapters/picoTracker/filesystem/picoTrackerFileSystem.cpp

```diff
@@ -5,14 +5,16 @@
 
 #include <string>
 
+#define MAGIC_PARENT_DIR_INDEX_MARKER 99999
```

Comment:
nit: wouldn't it be better for this to be 0 or -1?

Reply:
I wasn't sure if the dir indexes in SDFat can have a 0 value but yes, -1 is probably a better one here as I think these are signed ints already anyways.

Reply:
Yeah, let's change this before merging please.

Reply:
Done. changed to use `-1`